### PR TITLE
isosize: add page

### PR DESCRIPTION
**Pull request by en1gma713**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/isosize.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# isosize
+
+> Display the size of an iso file.
+
+- Display the size of an iso file:
```
Comment (navarroaxel) [inline pages/linux/isosize.md:5]:
```suggestion
- Display the size of an ISO file:
```

File: pages/linux/isosize.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`isosize {{iso file path}}`
```
Comment (navarroaxel) [inline pages/linux/isosize.md:7]:
```suggestion
`isosize {{path/to/file.iso}}`
```

File: pages/linux/isosize.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Diplay the block count and block size of an iso file:
+
+`isosize -x {{iso file path}}`
```
Comment (navarroaxel) [inline pages/linux/isosize.md:11]: We prefer the long argument format:
```suggestion
`isosize --sectors {{path/to/file.iso}}`
```

File: pages/linux/isosize.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Display the size of an iso file divided by a given number (only usable when -x is not given):
+
+`isosize -d {{number}} {{iso file path}}`
```
Comment (navarroaxel) [inline pages/linux/isosize.md:15]:
```suggestion
`isosize --divisor={{number}} {{path/to/file.iso}}`
```

File: pages/linux/isosize.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- Display the size of an iso file:
+
+`isosize {{iso file path}}`
+
+- Diplay the block count and block size of an iso file:
+
+`isosize -x {{iso file path}}`
+
+- Display the size of an iso file divided by a given number (only usable when -x is not given):
```
Comment (navarroaxel) [inline pages/linux/isosize.md:13]:
```suggestion
- Display the size of an ISO file divided by a given number (only usable when --sectors is not given):
```

File: pages/linux/isosize.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# isosize
+
+> Display the size of an iso file.
+
+- Display the size of an iso file:
+
+`isosize {{iso file path}}`
+
+- Diplay the block count and block size of an iso file:
```
Comment (navarroaxel) [inline pages/linux/isosize.md:9]: `block` or `sectors`?
I have this output:
```
$ isosize --sectors archlinux.iso
sector count: 348672, sector size: 2048
```
```suggestion
- Diplay the sector count and sector size of an ISO file:
```

File: pages/linux/isosize.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# isosize
+
+> Display the size of an iso file.
```
Comment (navarroaxel) [inline pages/linux/isosize.md:3]:
```suggestion
> Display the size of an ISO file.
```

**Review by navarroaxel — APPROVED** [review]
Hey @en1gma713, thanks for your submission. I left some comments.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
:man_facepalming: Did not meant to approve.

**Comment by en1gma713** [comment]
@navarroaxel in the man page it says that it displays block size but -x or --sectors uses the word sectors instead of blocks

**Comment by navarroaxel** [comment]
@en1gma713 you're right. Can we add a link to the man page?
```
> More information: <https://linux.die.net/man/8/isosize>.
```

File: pages/linux/isosize.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# isosize
+
+> Display the size of an iso file. More information: https://linux.die.net/man/8/isosize.
```
Comment (navarroaxel) [inline pages/linux/isosize.md:3]: We put the `more info` link in another line.
```suggestion
> Display the size of an ISO file.
> More information: <https://linux.die.net/man/8/isosize>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by sbrl — APPROVED** [review]
Looks ok to me when when @navarroaxel's comments are addressed

**Comment by en1gma713** [comment]
done

File: pages/linux/isosize.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+`isosize {{path/to/file.iso}}`
+
+- Diplay the block count and block size of an iso file:
+
+`isosize --sectors {{path/to/file.iso}}`
+
+- Display the size of an iso file divided by a given number (only usable when -x is not given):
+
+`isosize -d {{number}} {{path/to/file.iso}}`
```
Comment (navarroaxel) [inline pages/linux/isosize.md:16]:
```suggestion
`isosize --divisor={{number}} {{path/to/file.iso}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @en1gma713 the page is OK, you have only 3 minor changes.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**